### ISSUE TITLE
Issue with ReadyPlayerMe tablet.

### ISSUE THREAD

**Issue by Penguin-Guru** [vircadia/vircadia#1232]
I tried generating an avatar randomly (no image, no gender) and it worked well. I was able to customise as expected except for one issue. When I selected the clothes shown in the first picture, the tablet window turned entirely white and stayed that way for a minute or two until I closed the window. Re-opening the window by interacting with the tablet again behaved the same way. All clothing selections left of the one shown worked fine but this one did not. I did not test any farther to the right.

A day or two later, I tried again and I was able to put on the clothes with no issue (all of them). After closing the tablet window and re-opening it, I selected to view my avatar as before and the window is entirely grey (see second picture). This also repeated three times before I stopped testing.

I did not restart Vircadia between the tests in each set but I did restart between the two test sets. O.S. is Linux. Vircadia version is "Vircadia-2021.1.1-Eos.AppImage".

![cursorbox_h11m55s29](https://user-images.githubusercontent.com/22182988/120082110-f9d31900-c075-11eb-968e-e4425a9d2895.png)
![cursorbox_h11m57s12](https://user-images.githubusercontent.com/22182988/120082114-fb044600-c075-11eb-8c17-914639585d25.png)

**Comment by SilverfishVR** [comment]
works on windows (2021.1.2RC1) for me 🤔

**Comment by JulianGro** [comment]
Personally I have only ever seen the grey window for a second or two before stuff loaded in.
It would be worth to try the newest release candidate, since that includes the newest Qt version and with that the newest Qt Webengine version.
Specifically on AMD cards on Linux there were some rendering issues in the old Webengine which are fixed in the new version. http://appimage.moto9000.moe/release-candidate/Vircadia-x86_64_v2021.1.2-rc1.AppImage

**Comment by Penguin-Guru** [comment]
Ok, I'll give that a try but the issue does seem to be somehow related to that clothing selection. Just generating a new avatar (which I'm not sure how to do) might be interesting to try. This computer has an Intel processor and Nvidia graphics card but it's also about time I built a new one.

**Comment by Penguin-Guru** [comment]
I generated a random avatar using that release candidate and had no issues selecting any of the clothing options. I left it on the same clothing selection index that caused the problem previously and closed the window, reopened the window, and still experienced no trouble. That said, this avatar was obviously different, the clothes were all very different, I only tested the clothes, and I did not actually install the created avatar on my character. Could be that this release candidate fixes the issue, or it could have been some bug specific to the other avatar or clothing selection.

Side note, The Hub took about the same amount of time to render (old computer here) but the starting room rendered much faster than with the current stable.
